### PR TITLE
Increase chunk size in Data.stdout method

### PR DESCRIPTION
**Pull request by JureZmrzlikar**

### CODE REVIEW

**Comment by mzganec** [comment]
Locally I get the same error as displayed on Travis.

**Comment by dblenkus** [comment]
#circularimport

**Comment by dblenkus** [comment]
You should move the constant to a separate file.

**Comment by JureZmrzlikar** [comment]
Which file do you suggest, @dblenkus? `resdk.__init__.py`?

**Comment by dblenkus** [comment]
Something like `resdk/constants.py`.

**Comment by mstajdohar** [comment]
@dblenkus Please do another review pass.

File: resdk/resources/data.py [hunk 1]
```diff
@@ -307,13 +307,16 @@ def stdout(self):
         :rtype: string
 
         """
+        # Prevent circular imports:
+        from resdk.resolwe import CHUNK_SIZE
```
Comment (dblenkus) [inline resdk/resources/data.py]: Move `CHUNK_SIZE` to `constants.py` (or similar), so there will be no circular import problems.

**Pull request merged**